apolloschurchapp/src/client/index.js

```diff
@@ -74,21 +74,6 @@ export const client = new ApolloClient({
   shouldBatch: true,
   name: getApplicationName(),
   version: getVersion(),
-  // NOTE: this is because we have some very taxing queries that we want to avoid running twice
```

Comment:
@redreceipt did you confirm that this behavior is safe to remove? 

you would notice it when interacting with lists of content loaded in from the network. we added it since we were seeing far too many refetches when upgrading to apollo client 3